### PR TITLE
Add Github actions on top of PR #650

### PR DESCRIPTION
**Pull request by petergoldstein**
Add Github actions on top of PR #650

### CODE REVIEW

**Comment by petergoldstein** [comment]
@rposborne Travis.org is going away, so CI needs to transition to another system.  I threw together Github Actions, but for reasons that aren't clear they aren't running (essentially the same config is running on the logstasher repo)

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,10 @@
...
 language: ruby
 rvm:
- - 1.9.3
- - 2.3.6
- - 2.4.3
- - 2.5.0
- - ruby-head
+ - 2.5.8 
+ - 2.6.6
+ - 2.7,2
```
Comment (pmor) [inline .travis.yml:6]: You have a comma here, that's probably what's breaking travis

**Comment by pmor** [comment]
If I understand correctly, travis-ci.org will be replaced with travis-ci.com which continues to have a free plan for open source, but I can't find any details on what changes will be required or what limitations the new service might have, so I'm not sure we'll need to replace Travis.

Having said that, I have no objection with moving to GitHub actions

**Comment by petergoldstein** [comment]
@pmor Thanks for pointing out the comma.

Re: Travis, my understanding is that the situation is a little more ambiguous that that.  Either way, I'm not sure why Github Actions aren't running with the addition of a YAML file under `.github/workflows`

**Comment by petergoldstein** [comment]
@rposborne @pmor Do you have any idea when we'll see an new gem release for `countries`?

**Pull request merged**